### PR TITLE
Use tabs for create/load dataset steps

### PR DESCRIPTION
**Pull request by aidazolic**
Replaces expanders with tabs on the create new / load existing configuration page.

### CODE REVIEW

File: wizard/views/overview.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+
+def render_overview():
```
Comment (marcenacp) [inline wizard/views/overview.py:3]: If you rebase, you should be able to format using `black`.

**Review by marcenacp — APPROVED** [review]
LGTM, thanks!

Can you change the selector in the test (`[data-testid="stExpander"]`) with the new selector for tabs?

File: wizard/views/overview.py (hunk 1 above)
Comment (aidazolic) [reply wizard/views/overview.py:1]: Done

**Comment by aidazolic** [comment]
> LGTM, thanks!
> 
> Can you change the selector in the test (`[data-testid="stExpander"]`) with the new selector for tabs?

I modified the tests.
Can you PTAL again? Thanks!

**Review by marcenacp — APPROVED** [review]
Awesome, thanks!

**Pull request merged**